helpers/components/addInArticleAds.js

```diff
@@ -311,9 +317,10 @@ module.exports = function (content, articleData, featureTypes) {
       if (isTextComponent(content[index]) && !textComponentTooShort) {
         subsequent300x250Counter += 1;
 
-        if (isSurroundedByText(content, index) && subsequent300x250Counter % 5 === 0 && first300x250 && !isNearEndOfArticle(content, index) && getWordCount(content[index + 1]) > 85) {
+        if (isSurroundedByText(content, index) && subsequent300x250Counter % 5 === 0 && first300x250 && !isNearEndOfArticle(content, index)) {
```

Comment:
is removing the word count limit intentional - change shouldnt be made without product signoff? not seeing reference to this in the attached trello card.